datadog_lambda/trigger.py

```diff
@@ -0,0 +1,205 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+import base64
+import gzip
+import json
+from io import BytesIO, BufferedReader
+
+
+EVENT_SOURCES = [
+    "aws:dynamodb",
+    "aws:kinesis",
+    "aws:s3",
+    "aws:sns",
+    "aws:sqs",
+]
+
+GOVCLOUD_REGIONS = ["us-gov-east-1", "us-gov-west-1"]
+CHINA_REGIONS = ["cn-north-1", "cn-northwest-1"]
+
+
+def get_arn_region_identifier(region):
```

Comment:
nits: the formal name is partition, `get_aws_partition_by_region` https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html

Reply:
TIL!